lens/dict.rkt

```diff
@@ -0,0 +1,16 @@
+#lang racket/base
+
+(provide dict-ref-lens)
+
+(require racket/dict fancy-app "base/main.rkt")
+(module+ test
+  (require rackunit))
+
+(define (dict-ref-lens key)
+  (make-lens (dict-ref _ key)
+             (dict-set _ key _)))
+
+(module+ test
+  (check-equal? (lens-transform* '((a . 1) (b . 2) (c . 3)) (dict-ref-lens 'a) (* 100 _))
```

Comment:
I was thinking that `lens-transform*` makes more sense here because it has the same argument order as `dict-update`.